test/smtcallback.js

```diff
@@ -152,9 +162,13 @@ tape('SMTCheckerCallback', function (t) {
       st.comment('Collecting ' + sources[i] + '...');
       var source = fs.readFileSync(sources[i], 'utf8');
 
-      if (source.includes('// SMTEngine')) {
-        st.skip('Test requires specific SMTChecker engine.');
-        continue;
+      var engine;
+      if (source.includes('// SMTEngine:')) {
+        let re = new RegExp('// SMTEngine: (\\w+)');
+        let m = source.match(re);
+        assert(m !== undefined);
+        assert(m.length >= 2);
```

Comment:
If multiple engines are defined, they (second engine onwards) are silently ignored. Should we instead mandate single engine setting only or would that break something?

Reply:
@bshastry fixed

Reply:
and added tests for that